src/components/BorrowerProfile/BorrowerImage.vue

```diff
@@ -0,0 +1,136 @@
+<template>
+	<picture
+		class="
+			tw-inline-block
+			tw-relative
+			tw-overflow-hidden
+		"
+		:style="`padding-bottom: ${aspectRatio * 100}%;`"
+	>
+		<source :srcset="srcset" :sizes="sizes">
+		<img
+			v-if="hash"
+			class="
+				tw-absolute
+				tw-w-full
+				tw-h-full
+				tw-object-contain
+			"
+			:src="defaultUrl"
+			:alt="alt"
```

Comment:
a `loading="lazy"` attribute or `:loading="loading"` and using a prop would be good here.